client/dist/styles.css

```diff
@@ -0,0 +1,79 @@
+
+.app {
+  background-color: #F6F7F8;
+  float: left;
+  padding: 30px;
+}
+
+
+
+.attendees {
+  background-color: #F6F7F8;
+  font: "Graphik Meetup";
+}
+
+.attendee {
+  border-radius: 10px;
+  background-color: white;
+  font: "Graphik Meetup";
+  margin: 1px;
+  text-align: center;
+  padding: 5px 0px 0px 5px;
+  height: 150px;
+  width: 150px;
+}
+
+.attendee_picture {
+  margin-top: 10px;
+  border-radius: 100px;
+  height: 60%;
+  width: 60%;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+
+.attendees_header {
+  background-color: #F6F7F8;
+  display: inline-block;
+  padding-bottom: 20px;
+}
+
+.attendees_count {
+  display: inline-block;
+  font-family: Graphik Meetup,-apple-system,BlinkMacSystemFont,Roboto,Helvetica,Arial,sans-serif;
+  font-size: 20px;
+  text-decoration: bold;
+    background-color: #F6F7F8;
+}
+
+.seeAll {
+  display: inline-block;
+  background-color: #F6F7F8;
+  position: relative;
+  left: 450px;
+  color: #00a2c7;
+  font-family: Graphik Meetup,-apple-system,BlinkMacSystemFont,Roboto,Helvetica,Arial,sans-serif;
+  font-size: 16px;
+}
+
+.attendee:hover {
+  box-shadow: 1px 1px 1px 1px #C0C0C0;
+ }
+
+.seeAll:hover {
+  text-decoration: underline;
+}
+
+.attendee_name {
+  margin-top: 5px;
+  font-family: Graphik Meetup,-apple-system,BlinkMacSystemFont,Roboto,Helvetica,Arial,sans-serif;
+  font-size: 14px;
+}
+
+.attendee_role {
+  font-family: Graphik Meetup,-apple-system,BlinkMacSystemFont,Roboto,Helvetica,Arial,sans-serif;
```

Comment:
You list the font-family on several different div's.  If you put the font-family just once on the overarching app div, it should cascade down to all the children I believe.  Unless some other default font-family is overriding the font in the children?